pkg/vm/vm.go

```diff
@@ -1563,3 +1570,21 @@ func (v *VM) bytesToPublicKey(b []byte) *keys.PublicKey {
 	}
 	return pkey
 }
+
+// GetCallingScriptHash implements ScriptHashGetter interface
+func (v *VM) GetCallingScriptHash() util.Uint160 {
+	if v.Istack().Len() > 1 {
+		return v.GetContextScriptHash(1)
```

Comment:
I think we can hide `GetContextScriptHash()` now, it doesn't make a lot of sense and it's a bit dangerous to use.